server/src/YmlToBdl.g4

```diff
@@ -177,11 +169,11 @@ synonym: SYNONYM listWithBrace
         | SYNONYM OPEN_PAR (synonymElements+=value (COMMA synonymElements+=value)* )? CLOSE_PAR;
 
 //Question a Alain, est ce qu'on peut avoir autre chose qu'un 'field' comme memberType ?
-memberDeclaration : commentBlock? memberType memberName=ymlId memberOption=field*;
+memberDeclaration :  memberType memberName=ymlId memberOption=field*;
 path: ymlId (DOT ymlId)+?;
 ymlIdOrPath: ymlId | path;
 
-field : commentBlock? fieldArrow=(FIELD_INTRO | REPLACE_FIELD_VALUE_INTRO | REMOVE_FIELD | ADD_FIELD) optionName=ymlIdOrPath (optionValues+=valueOrCondition (COMMA optionValues+=valueOrCondition)*?)? ;

```

Comment:
Now, it is possible to write something like
```
--> domains /* I'm a comment. */ MyDomain
```

Before this commit, it wasn't possible because of the fixed position of the comment in the rule.